src/library/scala/runtime/LambdaDeserialize.java

```diff
@@ -2,28 +2,41 @@
 
 
 import java.lang.invoke.*;
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashMap;
 
 public final class LambdaDeserialize {
+    public static final MethodType DESERIALIZE_LAMBDA_MT = MethodType.fromMethodDescriptorString("(Ljava/lang/invoke/SerializedLambda;)Ljava/lang/Object;", LambdaDeserialize.class.getClassLoader());
+    public static final MethodType ADD_TARGET_METHODS_MT = MethodType.fromMethodDescriptorString("([Ljava/lang/invoke/MethodHandle;)V", LambdaDeserialize.class.getClassLoader());
```

Comment:
that's what you get pushing branches on 30 minutes of free wifi in Aukland airport 😄 

originally I thought that bootstrap arg list would be limited to 255 params (like regular param lists), but in fact it can be longer if the final param is an array type.